config/parser.go

```diff
@@ -186,18 +209,20 @@ type UIThemeConfig struct {
 type ThemeConfig struct {
 	Ui     UIThemeConfig     `yaml:"ui,omitempty"     validate:"omitempty"`
 	Colors *ColorThemeConfig `yaml:"colors,omitempty" validate:"omitempty"`
+	Icons  *IconThemeConfig  `yaml:"icons,omitempty" validate:"omitempty"`
 }
 
 type Config struct {
-	PRSections     []PrsSectionConfig    `yaml:"prSections"`
-	IssuesSections []IssuesSectionConfig `yaml:"issuesSections"`
-	Repo           RepoConfig            `yaml:"repo"`
-	Defaults       Defaults              `yaml:"defaults"`
-	Keybindings    Keybindings           `yaml:"keybindings"`
-	RepoPaths      map[string]string     `yaml:"repoPaths"`
-	Theme          *ThemeConfig          `yaml:"theme,omitempty" validate:"omitempty"`
-	Pager          Pager                 `yaml:"pager"`
-	ConfirmQuit    bool                  `yaml:"confirmQuit"`
+	PRSections      []PrsSectionConfig    `yaml:"prSections"`
+	IssuesSections  []IssuesSectionConfig `yaml:"issuesSections"`
+	Repo            RepoConfig            `yaml:"repo"`
+	Defaults        Defaults              `yaml:"defaults"`
+	Keybindings     Keybindings           `yaml:"keybindings"`
+	RepoPaths       map[string]string     `yaml:"repoPaths"`
+	Theme           *ThemeConfig          `yaml:"theme,omitempty" validate:"omitempty"`
+	Pager           Pager                 `yaml:"pager"`
+	ConfirmQuit     bool                  `yaml:"confirmQuit"`
+	ShowAuthorIcons bool                  `yaml:"showAuthorIcons"`
```

Comment:
I think this should also be defined in the section config.
This way, I can enable it globally but disable it for the "My PRs" section where it's irrelevant.

Reply:
Something like:

```yaml
prSections:
  - title: Work
    filters: is:open repo:work/project
    layout:
      authorIcon:
        hidden: true
```